Assets/AppCenter/Plugins/AppCenterSDK/Crashes/iOS/CrashesInternal.cs

```diff
@@ -107,6 +107,11 @@ public static ErrorReport GetErrorReportFromIntPtr(IntPtr errorReportPtr)
             return new ErrorReport(identifier, dtoStart, dtoError, exception, procId, reporterKey, reporterSignal, isAppKill, device);
         }
 
+        public static AppCenterTask<string> GetMinidumpDirectoryAsync() 
```

Comment:
Since the usage under `#if UNITY_ANDROID`, we can don't add it here

Reply:
Yeah, but we call an NDK method with this value, and if we do that on iOS it will crash

Reply:
Actually it still won't build on other platforms. Reverting this